### PR TITLE
Intellisense tokens made public

### PR DESCRIPTION
**Pull request by anderson-joyle**
Complement for https://github.com/microsoft/Power-Fx/issues/2161.

### CODE REVIEW

**Comment by LucGenetier** [comment]
✅ No public API change.

**Comment by MikeStall** [comment]
>     internal sealed class TokenTextSpan : ITokenTextSpan, ITextFirstFlag

Could we just make this class public?
(Public classes can have internal interfaces)



---
Refers to: src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/TokenTextSpan.cs:26 in b6fddcc. [](commit_id = b6fddcc9f4e5524ce47c8937788e8769ecac2593, deletion_comment = False)

**Comment by LucGenetier** [comment]
✅ No public API change.

File: src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/TokenTextSpan.cs [hunk 1]
```diff
@@ -23,7 +23,7 @@ internal interface ITokenTextSpan
         bool CanBeHidden { get; }
     }
 
-    internal sealed class TokenTextSpan : ITokenTextSpan, ITextFirstFlag
+    public sealed class TokenTextSpan : ITokenTextSpan, ITextFirstFlag
     {
         public string TokenName { get; private set; }
```
Comment (MikeStall) [inline src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/TokenTextSpan.cs:28]: >public string TokenName { get; private set; } [](http://example.com/codeflow?start=8&length=45)

what is this?

**Comment by LucGenetier** [comment]
✅ No public API change.

**Comment by LucGenetier** [comment]
✅ No public API change.

**Review by MikeStall — APPROVED** [review]

**Pull request merged**